bin/generateEasyConfig.R

```diff
@@ -59,35 +59,47 @@ repos$bioconductor$active = c('http://www.bioconductor.org/packages/release/bioc
 #    'NLopt'='2.6.1'
 #)
 #
-# This list is for R 4.0.x with foss 2018b:
+# This list is for R 4.2.x with foss 2022a:
 #
 dependency_defaults <-list(
-    'pkg-config'='0.29.2',
-    'libreadline'='8.0',
-    'ncurses'='6.2',
+    'pkgconf'='1.8.0',
+    'Xvfb'='21.1.3',
+    'Autotools'='20220317',
+    'R'='4.2.1-bare',
+    'X11'='20220504',
+    'Mesa'='22.0.3',
+    'libGLU'='9.0.2',
+    'cairo'='1.17.4',
+    'libreadline'='8.1.2',
+    'ncurses'='6.3',
     'bzip2'='1.0.8',
     'XZ'='5.2.5',
-    'zlib'='1.2.11',
-    'SQLite'='3.34.0',
-    'PCRE2'='10.36',
-    'Java'='11-LTS',
-    'cURL'='7.74.0',
-    'libxml2'='2.9.10',
+    'zlib'='1.2.12',
+    'SQLite'='3.38.3',
+    'PCRE2'='10.40',
     'libpng'='1.6.37',
-    'libjpeg-turbo'='2.0.6',
-    'LibTIFF'='4.2.0',
-    'cairo'='1.16.0-2',
-    'Pango'='1.48.0',
-    'OpenSSL'='1.1.1i',
+    'libjpeg-turbo'='2.1.3',
+    'LibTIFF'='4.3.0',
+    'Java'='11-LTS',
+    'Tcl'='8.6.12',
+    'Tk'='8.6.12',
+    'cURL'='7.83.0',
+    'libxml2'='2.9.13',
     'GMP'='6.2.1',
+    'NLopt'='2.7.1',
+    'FFTW'='3.3.10',
+    'libsndfile'='1.1.0',
+    'ICU'='71.1',
+    'HDF5'='1.12.2',
     'UDUNITS'='2.2.28',
-    'ImageMagick'='7.0.10-59',
-    'MariaDB-connector-c'='3.1.11',
```

Comment:
`MariaDB-connector-c` and `unixODBC` are required for interacting with MySQL/MariaDB databases. Required amongst other for fetching annotation directly from Ensembl databases. Why was it dropped?